### DISCUSSION TITLE
if i use air33 release chinses name app to ios , the name will be garbled , who can help me

### DISCUSSION THREAD

**Discussion by ifczt** [airsdk/Adobe-Runtime-Support#622] [Q&A]
air sdk is air 33.1.1.259

**Comment by ajwfrost** [comment]
Can you please confirm what values you're using in your application descriptor XML file please? or what process/IDE you're using for building the application?

We had an update a while ago that should have helped with this: #331 
so if you're able to provide more details on what you're using and what is then being generated, we can investigate further.

thanks

**Comment by ifczt** [comment]
@ajwfrost 
Thank you very much for your answer
im use utf-8

**Comment by ifczt** [comment]
<filename>智能学堂初中版</filename>
  <name>
    智能学堂初中版
  </name>
i set this app name but i get it '667a80fd5b665802521d4e2d7248' maybe is unicode cant use utf-8

**Comment by ajwfrost** [comment]
Where are you setting this name?
Do you edit the application descriptor XML file directly? or do you use Animate or similar?

thanks

**Comment by ifczt** [comment]
![QQ截图20201019173556](https://user-images.githubusercontent.com/4133892/96427803-92c4ca80-1231-11eb-8f38-7d3443420bc5.png)

yes im  edit the application descriptor XML file directly

**Comment by ifczt** [comment]
and im use flashdevelop

**Comment by ajwfrost** [comment]
Okay thanks. So, the issue is that the "filename" is set with Chinese characters. The filename is used for the "CFBundleExecutable" value and for the name of the generated executable file and .app folder: this causes problems for the signature process if it's got non-latin characters it seems.

So, if you can change the "filename" value to something more similar to your application ID or something else in Latin text, but leave the "name" value in Chinese, you should end up with a .ipa file that can be installed but then displays to the user with the Chinese name properly.

thanks

**Comment by ifczt** [comment]
sry if i change "filename" ios will report the ERROR ITMS-90694:" Invalid bundle display name."智能学堂初中版" starts with a whitespacecharacter cant change "filename" Is there any other solution

**Comment by ifczt** [comment]
@ajwfrost hi~ can u help me please i need u help ~~

**Comment by nyaaao** [comment]
@ifczt looking at your screenshot there's a space in the <name> in the beginning and that's why it fails. after you fix that it should work, I believe

**Comment by ifczt** [comment]
@nyaaao tanks but I tried, but it didn't work

**Comment by ifczt** [comment]
@ajwfrost Do you have any other solutions
